source/postprocess/particles.cc

```diff
@@ -651,6 +715,13 @@ namespace aspect
                              "move this file to a network file system. If this variable is "
                              "set to a non-empty string it will be interpreted as a "
                              "temporary storage location.");
+
+          prm.declare_entry ("Exclude output properties", "",
+                             Patterns::Anything(),
+                             "A comma seperated list of strings which exclude all particle"
+                             "property fields which contain these strings. If one of the "
+                             "entries is 'all', only a id will we provided for every point."
+                             "Todo: do we need a way to generate all posibilites?");
```

Comment:
It would be nice to document which options are available, but I do not see an easy way to do it. So do not worry about it for now.